### PR TITLE
Support for termination options for individual zones

### PR DESCRIPTION
**Pull request by TeresaRoberts**
Adds the option to add individual zones to your config file with different termination options, for example: manual, auto or the time to termination in minutes.

### CODE REVIEW

**Comment by TeresaRoberts** [comment]
The extra zones in the config file are compeltely optional and the plugin should works as before if they aren't configured. You only need to configure zones that don't follow the default behaviour. 

The code is tested on our system but apologies if the javascript isn't up to standard, it's been a while.

**Comment by lukasroegner** [comment]
First of all, thank you so much for the PR! I'll add my comments to the source code.

File: src/tado-heating-zone.js [hunk 1]
```diff
@@ -112,13 +112,31 @@ function TadoHeatingZone(platform, apiZone) {
     // Stores the contact sensor service
     zone.contactSensorService = contactSensorService;
 
+    // Sets termination variable from zone config
+    var terminationOption;
+    for (var i = 0; i < platform.config.zones.length; i++){
+        if (platform.config.zones[i].zoneId == zone.id){
+            break;
```
Comment (lukasroegner) [inline src/tado-heating-zone.js:120]: Shouldn't `terminationOption` bet set here before the `break`, e.g.

```
terminationOption = platform.config.zones[I]. terminationOption;
```

File: README.md [hunk 2]
```diff
@@ -51,9 +51,16 @@ npm install -g homebridge-tado-smart-thermostat
             "isApiEnabled": false,
             "apiPort": 40810,
             "apiToken": "<YOUR-TOKEN>"
```
Comment (lukasroegner) [inline README.md]: Comma missing here in the JSON, i.e.

```
"apiToken": "<YOUR-TOKEN>",
```

File: src/tado-heating-zone.js (hunk 1 above)
Comment (TeresaRoberts) [reply src/tado-heating-zone.js:120]: Apologies.  Somehow that line disappeared when updating the Github code.

File: README.md (hunk 2 above)
Comment (TeresaRoberts) [reply README.md]: Oops!

**Pull request merged**